strings.txt

```diff
@@ -31564,3 +31564,11 @@ fa = دوباره نپرس
 
   [subway_data_unavailable]
     ru = Карта метро недоступна
+
+  [bookmarks_empty_list_title]
+  	en = This list is empty
+    ru = Список пустой
+
+  [bookmarks_empty_list_message]
+  	en = To add a new place, tap on place and press the star icon
```

Comment:
To bookmark a place, tap it on the map and then tap the star icon
or
To add a bookmark, tap a place on the map and then tap the star icon